STPullToRefresh/STPullToRefreshHelper.m

```diff
@@ -77,6 +87,11 @@ - (void)setState:(STPullToRefreshState)state animated:(BOOL)animated {
     }
 }
 
+- (void)startRefreshAnimation:(BOOL)animated {
+    if (_state != STPullToRefreshStateLoading) {
```

Comment:
Good point, that check is redundant now. Will fix.

Previously needed to know whether to notify or not.